scripts/setup_hooks.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+git config core.hooksPath "./hooks"
```

Comment:
Yes, this could be added to the main setup script. In that case, we would also need to make the developer aware of the existence of the hook script and include an option (a flag) to disable this (in case he is using custom hooks of his own, or for any other reason).